### PR TITLE
Quickly correct all the mistake. And pull a new request. 

### PR DESCRIPTION
**Pull request by hanweifish**
Hopefully will not to delete the others person's file this time....
Correct:
1. check again, replace all the //global.mifile to http://global
2. check the tab and space. Yes, you are right. Thank.
3. reformat the note.html

PS: This note.html is formed with the note I took on the class by the JavaScript, since I am lazy....

### CODE REVIEW

File: Wei/03 CSS/formatted note.html [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+            <hr>
+            <li>shopify: based on nodejs</li>
+            <hr>
+            <li>Homework</li>
+            <hr>
+            <li>1. 仿站结果上传ftp，看看效果</li>
+            <hr>
+            <li>2. 熟悉wordpress</li>
+            <hr>
+            <li>3. 简历模板，选一个style，从头开始， 发pull request</li>
```
Comment (zizuoliu) [inline Wei/03 CSS/formatted note.html:120]: 你用的是windows吧，我用mac打开，中文字都是乱码

Comment (zizuoliu) [inline Wei/03 CSS/formatted note.html:120]: 需要在head里预处理charset，加这么一行 
meta charset="utf-8"

Comment (hanweifish) [inline Wei/03 CSS/formatted note.html:120]: 恩，加了以后就work了
<meta charset="UTF-8”>

**Comment by hanweifish** [comment]
https://github.com/hanweifish/legoclicker https://github.com/hanweifish/legoclicker

**Pull request merged**